DEVELOP.rst

```diff
@@ -10,28 +36,29 @@ directory:
 
 .. code-block:: console
 
-   $ mkdir -pv ${GOPATH}/src/github.com/crate
-   $ cd ${GOPATH}/src/github.com/crate
-   $ git clone https://github.com/crate/cratedb-prometheus-adapter.git
-   $ cd cratedb-prometheus-adapter
+   mkdir -pv ${GOPATH}/src/github.com/crate
```

Comment:
If you remove the `$` prefix in the code block, you can also remove the `console` syntax highlighting. The point of the `$` is, that these lines are treated as "input lines", whereas lines without the prefix are "output lines", and the syntax highlightin would distinguish between them.

Reply:
Thank you!